File: Aula 2/Aula2.dart
class PesoAltura {
  final int peso;
  final double altura;

  PesoAltura(this.peso, this.altura);
}

var pessoa1 = PesoAltura(70, 1.75);
var alturaMaxima = 1.8;
void main() {
  if (pessoa1.altura > alturaMaxima) {
    print('Peso ${pessoa1.peso} e Altura  ${pessoa1.altura} > $alturaMaxima');
  } else {
    print('Peso ${pessoa1.peso} e Altura  ${pessoa1.altura} < $alturaMaxima');
  }
}

File: Aula 2/Aula2.js
class PesoAltura {
    constructor(peso, altura) {
        this.peso = peso;
        this.altura = altura;
    };

}
let alturaMaxima = 1.8
var pessoa1 = new PesoAltura(70, 1.85)
if (pessoa1.altura > alturaMaxima) return 'Ultrapassou a altura permitida'
console.log(pessoa1)